src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -190,6 +181,25 @@ void Terminal::UpdateSettings(ICoreSettings settings)
     // remains at the bottom of the buffer.
 }
 
+// Method Description:
+// - Update our internal properties to match the new values in the provided
+//   CoreAppearance object.
+// Arguments:
+// - appearance: an ICoreAppearance with new settings values for us to use.
+void Terminal::UpdateAppearance(ICoreAppearance appearance)
+{
+    // Set the default background as transparent to prevent the
+    // DX layer from overwriting the background image or acrylic effect
+    til::color newBackgroundColor{ static_cast<COLORREF>(appearance.DefaultBackground()) };
+    _defaultBg = newBackgroundColor.with_alpha(0);
+    _defaultFg = appearance.DefaultForeground();
+

```

Comment:
Unfortunately, to set the style of the cursor we also need the cursor height, which is currently not in `ICoreAppearance` because it could potentially cause a resize and we decided we are not going to allow that for now.